### PR TITLE
Update templates/LABELS.md

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

This file is an additional maintenance burden with no real use.

**Release note**:

```release-note
None
```

### CODE REVIEW

File: templates/LABELS.md [hunk 1]
```diff
@@ -1,97 +0,0 @@
-# Available labels
-
-This file describes the labels we will be using to select
-the workload a template is supposed to be used for.
-
-Some of them are not going to be used immediately and serve
-as examples.
-
-## Deprecated templates
```
Comment (ksimon1) [inline templates/LABELS.md:9]: delete only this paragraph, not whole file

**Comment by yaacov** [comment]
> @yaacov Could you have a look at this? Is this file still needed by UI?

It's just the documentation right ?

if it's not just documentation:
the UI use `deprecated` label, to not show it to users
the UI use the `os.template.kubevirt.io`, to guess the os name
workload and flavor are used to desplay nice information about the template

we need this labels in 4.10
we can deprecate them in 4.11, are the getting deprecated ?

**Comment by 0xFelix** [comment]
@yaacov Yes, it is only documentation. Sorry for the misunderstanding. It is tedious to update the versions in `LABELS.md` manually. The labels are definitely going to stay.

I updated the PR so LABELS.md is kept and changed the tedious parts instead.

**Pull request merged**